src/shared/components/query/MolecularProfileSelector.tsx

```diff
@@ -26,11 +27,15 @@ const styles = styles_any as {
 @observer
 export default class MolecularProfileSelector extends QueryStoreComponent<{}, {}>
 {
+	private get showGSVA() {

```

Comment:
We should only interface with configuration via AppConfig.  So this would be AppConfig.serverConfig.skin_show_gsva.  
This MIGHT make it harder to change configuration dynamically in test.  It could still work the way you have it in the spec up above.

Reply:
@alisman Thanks. I have updated the PR. So when should we use `getBrowserWindow()`?

Reply:
ideally never!  it's a global and should be avoided.   it's in the code base as a convenience to avoid having to pass, say, the routing store into the view layers.  but really it's not a good practice.   if you for some reason you find yourself needing to use it, let me know.